scripts/interpreter.js

```diff
@@ -23,11 +23,11 @@
             if (speed > 100) speed = 100;
             if (speed < -100) speed = -100;
             switch (motor) {
-                case 'motorA':
+                case 'motorLeft':
```

Comment:
motorA, B, C are on purpose - this is what students are used to in Lego software.